src/aws-cpp-sdk-core/include/smithy/client/AwsSmithyClientBase.h

```diff
@@ -65,6 +67,33 @@ namespace client
     /* Non-template base client class that contains main Aws Client Request pipeline logic */
     class SMITHY_API AwsSmithyClientBase
     {
+    private:
```

Comment:
lets move private definitions after public per the [google style guide](https://google.github.io/styleguide/cppguide.html#Declaration_Order)

> A class definition should usually start with a public: section, followed by protected:, then private:. Omit sections that would be empty.